vip-workflow.php

```diff
@@ -53,3 +53,4 @@
 // Modules
 require_once VIP_WORKFLOW_ROOT . '/modules/editorial-metadata/editorial-metadata.php';
 require_once VIP_WORKFLOW_ROOT . '/modules/preview/preview.php';
+require_once VIP_WORKFLOW_ROOT . '/modules/notifications/notifications.php';
```

Comment:
Can we make these alphabetical per section? No real purpose other than consistency.

Reply:
Done